FirebaseAuth/Tests/SampleSwift/AuthenticationExample.xcodeproj/project.pbxproj

```diff
@@ -507,8 +510,8 @@
 			buildActionMask = 2147483647;
 			files = (
 				EA217895248979E200736757 /* LaunchScreen.storyboard in Resources */,
-				8848765129D314A400780FA6 /* GoogleService-Info.plist in Resources */,
```

Comment:
the failed CI seems related to the plist file, can you revert the plist file part in this file and then try trigger CI again?